pkg/cloud/vsphere/services/userdata/controlplane.go

```diff
@@ -209,9 +246,20 @@ type ContolPlaneJoinInput struct {
 	SaKey             string
 	BootstrapToken    string
 	ELBAddress        string
+	CloudConfig       string
 	JoinConfiguration string
 }
 
+type CloudConfigInput struct {
```

Comment:
We need to add a comment to this exported type. I don't want to make reconciling things to where we can enable listing worse. However, since this is passing CI for now, let's get back to it. I don't want to chance Prow today.